src/cmd/setup-tachyon.js

```diff
@@ -132,6 +134,18 @@ module.exports = class SetupTachyonCommands extends CLICommandBase {
 		return deviceId;
 	}
 
+	async _getDeviceInfo() {
+		try {
+			return await this.ui.showBusySpinnerUntilResolved('Getting device info', getTachyonInfo({
+				outputLog: this.outputLog,
+				ui: this.ui,
+			}));
+		} catch (error) {
+			// ignore error and return default values
+			this.ui.write('We couldn\'t get the device info.');
```

Comment:
I'm hesitating on returning here all the properties as `Unknown` or just catch the `undefined` as I'm doing right now